frontend/src/resources/LoginForm.css

```diff
@@ -0,0 +1,21 @@
+.input-field {
+    margin-bottom: 2%;
+}
+
+.input-container {
+    align-items: center;
```

Comment:
Is this line needed

Reply:
This container is needed for positioning the input fields in the center.